src/main/cpp/utilities/debug.cpp

```diff
@@ -22,9 +22,13 @@
  */
 #include "debug.h"
 #include "../global.h"
-#include <android/log.h>
 #include <string>
 
+#ifdef DEBUG
+#include <android/log.h>
+#include <stdio.h>
+#endif
+
```

Comment:
Don't include debug code requirements if the DEBUG definition has been commented in _global.h_